### PR TITLE
Add back compatibility with IDA 7.0 (Python 2)

### PR DESCRIPTION
**Pull request by p-state**
This PR depends on #46, so merge that first.
Didn't test it well, may have bugs.

### CODE REVIEW

File: HexRaysPyTools/callbacks/renames.py [hunk 1]
```diff
@@ -228,7 +228,7 @@ def __can_be_part_of_assert(cfunc, ctree_item):
 
         obj_ea = expression.obj_ea
         if not helper.is_code_ea(obj_ea) and idc.get_str_type(obj_ea) == idc.STRTYPE_C:
-            str_potential_name = idc.get_strlit_contents(obj_ea)
+            str_potential_name = str(idc.get_strlit_contents(obj_ea).decode('latin-1'))
```
Comment (igogo-x86) [inline HexRaysPyTools/callbacks/renames.py]: Latin? Is it really necessary?

Comment (p-state) [reply HexRaysPyTools/callbacks/renames.py]: actually ASCII is enough here, fixed

File: HexRaysPyTools/callbacks/actions.py [hunk 2]
```diff
@@ -65,7 +65,9 @@ def activate(self, ctx):
         # type: (idaapi.action_activation_ctx_t) -> None
         raise NotImplementedError
 
-    def check(self, hx_view: idaapi.vdui_t):
+    def check(self, hx_view):
+        """hx_view: idaapi.vdui_t
+        """
```
Comment (igogo-x86) [inline HexRaysPyTools/callbacks/actions.py]: One more thing - there's python2 type [hinting style](https://www.python.org/dev/peps/pep-0484/) which for example PyCharm understands. So it should be:

```python
    def check(self, hx_view):
       # type: (hx_view: idaapi.vdui_t) -> bool
```

**Comment by MegaBor** [comment]
Awesome job! Thanks a lot!

**Pull request merged**